frontend/packages/noobaa-storage-plugin/integration-tests/views/createBC.view.ts

```diff
@@ -0,0 +1,159 @@
+import { execSync } from 'child_process';
+import * as _ from 'lodash';
+import { $, $$, browser, by, element, ExpectedConditions as until } from 'protractor';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+
+// Backing Store is the first one in Operator Hub
+export const createBucketClassLink = $('article:nth-child(3) a');
+const tier1TableCheckBoxes = $$('table[aria-label="Select Backing Store for Tier 1"] input');
+const tier2TableCheckBoxes = $$('table[aria-label="Select Backing Store for Tier 2"] input');
+
+// Wizard Controls
+const nextButton = element(by.buttonText('Next'));
+const createBucketClassButton = element(by.buttonText('Create Bucket Class'));
+const bucketClassNameInput = $('input[aria-label="Bucket Class Name"]');
+const bucketClassDescriptionInput = $('textarea[aria-label="Description of bucket class"]');
+
+// Tier Buttons
+const tier1Spread = $('input#radio-1');
+const tier1Mirror = $('input#radio-2');
+const tier2Spread = $('input#radio-3');
+const tier2Mirror = $('input#radio-4');
+const addTierButton = $('button[data-testid="add-tier-btn"]');
+// Review Page Data
+const bcName = $('p[data-testid="bc-name"]');
+const bcNamespace = $('p[data-testid="namespace-value"]');
+const bcDescription = $('p[data-testid="bc-desc"]');
+const tier1Policy = $('h5[data-testid="tier1-policy"]');
+const tier2Policy = $('h5[data-testid="tier2-policy"]');
+
+export enum Tier {
+  SPREAD = 'SPREAD',
+  MIRROR = 'MIRROR',
+}
+
+const TierCountMap = Object.freeze({
```

Comment:
```suggestion
const TierBackingStoreMap = Object.freeze({
```

Reply:
TierCountMap is a more meaningful IMO. It serves as a map to link a tier name to the count of stores needed.